1.5/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
   && wget -O /tmp/node-v6.6.0-linux-x64.tar.xz https://nodejs.org/dist/v6.6.0/node-v6.6.0-linux-x64.tar.xz \
   && wget -O /tmp/node-v6.9.3-linux-x64.tar.xz https://nodejs.org/dist/v6.9.3/node-v6.9.3-linux-x64.tar.xz \
   && wget -O /tmp/node-v6.10.2-linux-x64.tar.xz https://nodejs.org/dist/v6.10.2/node-v6.10.2-linux-x64.tar.xz \
```

Comment:
Hmm I just looked at your changes again and I misinterpreted them due to the way github renders the diff. @naziml was that the intent of your comment? If so, he needs to remove more than just the wget line.